app/templates/screen/Presenter.kt.mustache

```diff
@@ -36,14 +36,14 @@ class {{screenUpperCamel}}Presenter(private val context: Context) {
     }
 
     fun onResume() {
-        subscriptions = RxUtils.getNewCompositeSubIfUnsubscribed(subscriptions)
+        subscriptions = CompositeSubscription()
         if (!state.initialized) {
             state.initialized = true
         }
     }
 
     fun onPause() {
-        RxUtils.unsubscribeIfNotNull(subscriptions)
```

Comment:
Non nullable subscription means we don't need a util call here. (Even if we did I'd prefer a `?` call anyway)